pyproject.toml

```diff
@@ -1,7 +1,168 @@
 [build-system]
-requires = ["setuptools>=67", "setuptools_scm[toml]>=7.0"]
+requires = [
+    "setuptools==69.0.0",
+    "setuptools_scm==8.0.4",
+]
 build-backend = "setuptools.build_meta"
 
+[project]
+dynamic = ["version"]
+name = "briefcase"
+description = "Tools to support converting a Python project into a standalone native application."
+readme = "README.rst"
+requires-python = ">= 3.8"
+license.file = "LICENSE"
```

Comment:
I know you're pretty steeped in this and have probably considered the aspects around this.

But do you know how this will be displayed in PyPI? Curious because currently it at least says "New BSD". Unfortunately, the available trove classifier remains ambiguous with just "BSD License".

I didn't go all the way down the rabbit hole but I think [PEP-639](https://peps.python.org/pep-0639/) finally added support to explicitly say we're using bsd-3-clause.

Reply:
I hadn't contemplated PyPI presentation. From the look of some high profile packages that are using PEP621, ([attrs](https://pypi.org/project/attrs/), [cryptography](https://pypi.org/project/cryptography/), [setuptools_scm](https://pypi.org/project/setuptools-scm/)), they're using `license.text` (or just `license`), not `license.file`; so that might be the better way to go. Looks like this approach puts the same License marker in the METADATA, and just having a LICENSE file in the repo is enough for wheels to include that file and include it as license content.

PEP639 would definitely alter this, but it is still in draft.

My survey of projects also pointed out that the way I handled the "Homepage" link isn't quite right. Fixes incoming.

Reply:
Uploaded (with these changes) here: https://test.pypi.org/project/briefcase-rmartin16/0.3.17.dev112/

Reply:
All looks suitably pretty to me...

Reply:
Yeah....they don't seem to respect ordering of URLs, though.... 😐 🤷🏼‍♂️

Reply:
Alphabetical is *an* ordering... :-)